mflowgen/full_chip/construct.py

```diff
@@ -419,9 +436,8 @@ def construct():
   order.insert( index, 'netlist-fixing.tcl' )
   signoff.update_params( { 'order': order } )
 
-  # Fill adds _filled to end of design_name
-  drc.update_params( { 'design_name': parameters['design_name'] + "_filled" } )
-  antenna_drc.update_params( { 'design_name': parameters['design_name'] + "_filled" } )
+  merge_rdl.update_params( {'coord_x': '613.565u', 'coord_y': '3901.872u', 'flatten_child': True} )
```

Comment:
Can we move these exact numbers up to the top of the file so they're more easily found/adjusted as needed?

Reply:
or use actual mflowgen params